nautobot/dcim/forms.py

```diff
@@ -786,6 +786,24 @@ class ManufacturerFilterForm(NautobotFilterForm):
     platforms = DynamicModelMultipleChoiceField(queryset=Platform.objects.all(), to_field_name="name", required=False)
 
 
+#
+# Platform
+#
+class PlatformBulkEditForm(NautobotBulkEditForm):
+    pk = forms.ModelMultipleChoiceField(queryset=Platform.objects.all(), widget=forms.MultipleHiddenInput())
+    description = forms.CharField(max_length=CHARFIELD_MAX_LENGTH, required=False)
+    name = forms.CharField(max_length=CHARFIELD_MAX_LENGTH, required=False)
+    manufacturer = forms.ModelChoiceField(
+        queryset=Manufacturer.objects.all(), required=False, widget=forms.Select(attrs={"class": "form-control"})
+    )
+    network_driver = forms.CharField(max_length=CHARFIELD_MAX_LENGTH, required=False)
+    napalm_driver = forms.CharField(max_length=CHARFIELD_MAX_LENGTH, required=False)
+    napalm_args = forms.JSONField(required=False, widget=forms.Textarea(attrs={"rows": 4, "class": "form-control"}))
```

Comment:
The `form-control` class should be applied automatically by the base form class, it shouldn't need to be explicitly applied here.